### ISSUE TITLE
Create Variables for heterogenous treatment effects

### ISSUE THREAD

**Issue by Jo-Schie** [openkfw/mapme.protectedareas#97]
- [ ] size of protected area (WDPA)
- [ ] Designation and Indigenous People: z.B DESIG_ENG (Designation in WDPA) -> see [this study](https://www.sciencedirect.com/science/article/abs/pii/S000632072200026X )
- [ ] treatment type kfw (requires internal documentation)

**Comment by yotaae** [comment]
I have created a markdown file to check the different categories in the WDPA database (see first point in comment above). 

As expected, some of the variables (e.g. OWN_TYPE) have "not reported" or "not applicable" in many cases, which is why they are not really useful for an analysis of heterogeneous treatment effects. 

The variables GOV_TYPE, IUCN_CAT (here, we have some note reported, too), ecosystem_kfw or DESIG_TYPE (not sure what this refers to) look more promising. 

Other variables like type_kfw or DESIG_ENG have (in my opinion) too many categories. 

See file here:  /datadrive/yota/development/WDPA_heterogeneity_vars.html

**Comment by Jo-Schie** [comment]
as for internal documentation: I suggested this methodology internally: [https://conbio.onlinelibrary.wiley.com/doi/epdf/10.1111/j.1523-1739.2008.00937.x](https://conbio.onlinelibrary.wiley.com/doi/epdf/10.1111/j.1523-1739.2008.00937.x)

Please have a look at it and let's discuss.

**Comment by yotaae** [comment]
@Jo-Schie Good point, I'll add the files to main. Had some issues with pushing the files this morning but I'll update you here as soon as I have worked it out. 

Regarding IUCN categories: we do not have missing values for IUCN but we have "Not reported" (n=28), "not applicable" (n=3) and "not assigned" (n=3) in some cases. Total number of PAs should be 416 if I haven't made any mistakes. Can we work with the above categories? 

Regarding internal data: I have merged our assetids with bmz numbers and our project data. I still have to check the data though, since some assetids seem to be assigned to multiple WDPA IDs. Can't say if that's a problem for now - will update here.

**Comment by Jo-Schie** [comment]
For me that seems to be a good variable. We should use it and quantify results. I would use "not reported" and similar as categories rather then missings @yotaae and @melvinhlwong

**Comment by melvinhlwong** [comment]
thanks for the info
